frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -305,6 +313,12 @@ export enum MemoryUnits {
   Gi = 'Gi',
 }
 
+export enum AutoscaleWindowUnits {
+  s = 'Sec',
+  m = 'Min',
+  h = 'Hrs',
+}
```

Comment:
I think you'll need to localize the units here.
cc @openshift/team-devconsole-ux

Reply:
localize the units.